### PR TITLE
Operator 2.4.0

### PR DESCRIPTION
**Pull request by ryanemerson**

### CODE REVIEW

File: _posts/2024-03-19-infinispan-operator-2-4-0-release.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+Our intention is to continue our current versioning scheme, whereby the Operator minor version is incremented when support
+for a new Infinispan major version is added. However, to simplify upgrades, all future Operator releases will be added to
+the "stable" channel, instead of `x.y.z`, so that it's no longer necessary for the Subscription channel to be changed when
+upgrading.
+
+== Infinispan Server Support
+
+Future 2.4.x Operator releases will include the latest releases for both the 15.0.x and 14.0.x Infinispan server streams.
+Whilst we always recommend users to transition to the latest Infinispan major version as soon as possible, supporting both
```
Comment (karesti) [inline _posts/2024-03-19-infinispan-operator-2-4-0-release.adoc:43]: While

File: _posts/2024-03-19-infinispan-operator-2-4-0-release.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+for Kubernetes and Red Hat OpenShift.
+
+This is the first Operator release based on Infinispan 15.
+
+== Release highlights:
+
+* Infinispan 14.0.27.Final support
+* Infinispan 15.0.0.Final support
+* Allow Readiness, Liveness and Startup probe values to be configured
+* Simplify JGroups stacks
```
Comment (karesti) [inline _posts/2024-03-19-infinispan-operator-2-4-0-release.adoc:23]: I'll add this one on the end

Could we add some more explanations to each part ?

File: _posts/2024-03-19-infinispan-operator-2-4-0-release.adoc [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+* CloudEvents integration no longer has any effect with 15.0.x servers onwards
+* XSite improvements:
```
Comment (karesti) [inline _posts/2024-03-19-infinispan-operator-2-4-0-release.adoc:25]: XSite does not mean much for many people. Could we use Cross-Site Replication instead and put XSite like (XSite)?

**Review by karesti — CHANGES_REQUESTED** [review]

**Comment by ryanemerson** [comment]
Updated, thanks @karesti

**Review by karesti — APPROVED** [review]

**Pull request merged**